src/server/routes/api/document/index.js

```diff
@@ -178,6 +180,66 @@ const DEFAULT_CORRESPONDENCE = {
   emails: []
 };
 
+const mapToUniprotIds = docTemplate => {
+  const updateGrounding = entityTemplate => {
+    if ( entityTemplate == null ) {
+      return Promise.resolve();
+    }
+
+    let xref = entityTemplate.xref;
+    let { db, id } = xref;
+
+    if ( db !== 'NCBI Gene' ){
+      return Promise.resolve();
+    }
+
+    const opts = {
+      id: [
+        id
+      ],
+      dbfrom: 'ncbigene',
```

Comment:
See comment above in entity.js about using `dbPrefix` in the xref.db field. That way, we we're not making up names for dbs.